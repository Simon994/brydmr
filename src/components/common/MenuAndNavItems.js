import React from 'react'
import { NavLink, withRouter } from 'react-router-dom'

const MenuAndNavItems = (props) => {
  const [listItemOne, listItemTwo] = [...props.listItems]

  return (
    <ul>
      <li>
        <a href='#'>
        {listItemOne.toUpperCase()}
        </a>
      </li>
      <li>
        <a href='#'> 
          exact to={`/${listItemTwo}`}
          className='nav-link'
          activeClassName='selected'
        >
        {listItemTwo.toUpperCase()}
        </a>
      </li>
    </ul>
  )
}

export default MenuAndNavItems