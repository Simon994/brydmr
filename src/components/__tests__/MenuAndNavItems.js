import React from 'react'
import renderer from 'react-test-renderer'
import MenuAndNavItems from '../common/MenuAndNavItems'


describe('The Menu', () => {
  jest.mock('../../../node_modules/react-router-dom/NavLink', () => () => '<nav/>')
  it('renders as expected', () => {
    const tree = renderer.create(<MenuAndNavItems listItems={['store', 'tutorials']}/>).toJSON()

    expect(tree).toMatchSnapshot()
  })
})