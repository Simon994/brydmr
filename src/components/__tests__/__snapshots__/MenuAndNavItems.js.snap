// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Menu renders as expected 1`] = `
<ul>
  <li>
    <a
      href="#"
    >
      STORE
    </a>
  </li>
  <li>
    <a
      href="#"
    >
      exact to=
      /tutorials
      className='nav-link' activeClassName='selected' &gt;
      TUTORIALS
    </a>
  </li>
</ul>
`;
